From 3df5bdd129bc76c94567eb36a3e2cb870b023422 Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Thu, 23 Sep 2021 10:36:06 +1000
Subject: [PATCH] wininet: Allow up to 4K for data buffers

LevelHead includes a Header value, that is > 3K in length which
causes a crash after linking your account.
---
 dlls/wininet/internet.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/wininet/internet.h b/dlls/wininet/internet.h
index e9d68e2c2d9..8538699811c 100644
--- a/dlls/wininet/internet.h
+++ b/dlls/wininet/internet.h
@@ -462,7 +462,7 @@ void free_authorization_cache(void) DECLSPEC_HIDDEN;
 
 void init_winsock(void) DECLSPEC_HIDDEN;
 
-#define MAX_REPLY_LEN	 	0x5B4
+#define MAX_REPLY_LEN	 	0x1000
 
 /* Used for debugging - maybe need to be shared in the Wine debugging code ? */
 typedef struct
-- 
2.33.0

